src/pg_cron.c

```diff
@@ -1267,5 +1561,536 @@ ManageCronTask(CronTask *task, TimestampTz currentTime)
 			InitializeCronTask(task, jobId);
 		}
 
+	/* keep compiler quiet */
+	(void) result;
 	}
 }
+
+static void
+GetTaskFeedback(shm_mq_handle *responseq, PGresult *result, CronTask *task)
+{
+
+	TimestampTz end_time;
+	bool update_result;
+
+	end_time = GetCurrentTimestamp();
+
+	if (responseq) {
```

Comment:
it would probably look a little cleaner if we just had two separate GetTaskFeedback functions, given that almost nothing is shared between the two cases.